Script/AtomicEditor/ui/modal/About.ts

```diff
@@ -90,7 +90,7 @@ class About extends ModalWindow {
         else
             availableText += "<widget TBSkinImage: skin: 'LogoAndroid-Small'> ";
 
-            if (licenseSystem.licenseIOS)
+            if (licenseSystem.licenseIos)
```

Comment:
Man, iOS is so gnarly, every CamelCase always looks wrong with it.  This probably should be licenseIOS though, licenseIos looks pretty weird and is difficult to "see".